contrib/tig/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "tig"
+pkgver = "2.5.8"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = []
```

Comment:
any reason the autotools cannot be regenerated?

Reply:
No reason, configure was already present so was making use of it but if the done thing is to regenerate it then I'm happy to make that change.